pkg/sql/sql.y

```diff
@@ -82,7 +82,7 @@
 
 	/* If no FOR in the COLUMN, the key is "" */
 	type columnClause map[string]exprlist
-	type filedClause  exprlist
+	type fieldClause  exprlist
```

Comment:
It seems the old `filedClause` is  useless.

Reply:
Yes, I keep it just in case. Maybe we need another commit to clean such stale syntax rules.